prow/README.md

```diff
@@ -184,3 +184,23 @@ GitHub webhook needs to be configured with a payload URL pointing to the hook se
 Add the URL and token as below. Select **"Send me everything"**, and for Content ype: **application/json**.
 
 ![webhook](images/webhook.png)
+
+## Enabling metal3 prow for new org/repo
+
+metal3 prow is currently working with two github organizations(orgs): `metal3-io` and `Nordix`. For `Nordix` we have enabled prow only for two repositories, namely: metal3-dev-tools and metal3-clusterapi-docs. We don't forsee enabling metal3 prow for any other github org, however we might need to enable prow in other repositories in existing github orgs for example . In any case we should follow the steps below to enable prow:
+
+1. Add/check `metal3-io-bot` user in the github org with `admin` access. Check the image 
+
+![](images/metal3-io-bot.png)
+
+2. Enable prow webhook as described in [GitHub webhook configuration](#github-webhook-configuration) section. For `metal3-io` the webhook is enabled in org level. For the two repositories in `Nordix` org we have enabled them on individual repositories. Keep in mind that the hmac token and hook URL is the same as described in [GitHub webhook configuration](#github-webhook-configuration). The webhook should look happy (green tik) as shown in the image below once you have configured correctly and communication has been established between github and prow hook. 
```

Comment:
- hmac -> HMAC.
- token and hook URL is -> token and hook URL are
- tik -> tick
- configured correctly -> configured it correctly
- github -> GitHub